requirements.txt

```diff
@@ -6,6 +6,12 @@ h5py~=2.10.0
 Keras==2.3.1
 ivadomed
 matplotlib
+# Fresh Windows installations may be missing the C++ runtime library required by scikit-image.
+# Installing "Microsoft Visual C++ Redistributable for Visual Studio" will fix this too, but
+# we'd prefer not to burden users with another prerequisite. 
+# NB: It's possible this fix gets included upstream in a future skimage release, see:
+#     https://github.com/scikit-image/scikit-image/issues/4780#issuecomment-824678872
+msvc-runtime; sys.platform == 'win32'
```

Comment:
Ah, interesting -- this doesn't actually represent an importable module of any kind, which our `sct_check_dependencies` doesn't like:

```
 Check if msvc-runtime is installed..................[FAIL]
An error occured while importing module msvc-runtime -> No module named 'msvc-runtime'
Full traceback: Traceback (most recent call last):
  File "d:\a\spinalcordtoolbox\spinalcordtoolbox\spinalcordtoolbox\spinalcordtoolbox\scripts\sct_check_dependencies.py", line 277, in main
    module = module_import(module_name, suppress_stderr)
  File "d:\a\spinalcordtoolbox\spinalcordtoolbox\spinalcordtoolbox\spinalcordtoolbox\scripts\sct_check_dependencies.py", line 110, in module_import
    module = importlib.import_module(module_name)
  File "C:\hostedtoolcache\windows\Python\3.7.9\x64\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 965, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'msvc-runtime'
```

I'll poke around to see if there's a nice solution for this.

Reply:
Oh, this is just because `msvc-runtime` needs to map to `msvc_runtime`. :P